test/cmdlineTests/model_checker_contracts_all/err

```diff
@@ -6,10 +6,10 @@ x = 0
 Transaction trace:
 B.constructor()
 B.f(0)
- --> model_checker_contracts_all/input.sol:5:3:
+ --> model_checker_contracts_all/input.sol:5:9:
   |
-5 | 		assert(x > 0);
-  | 		^^^^^^^^^^^^^
+5 |         assert(x > 0);
+  |         ^^^^^^^^^^^^^
```

Comment:
Note that tabs here come from the input file, which is indented with spaces. SMT checker sets the location to the whole line, not just the assert so the snippet includes indentation.

It could be worked around by changing the indentation to use spaces to match our style for Solidity, like I did here, but it's too tedious so I did not do that for other files. We should really fix SMTChecker to match the behavior of normal analysis and not include indents in locations.

Reply:
Maybe we should create an issue for that?

Reply:
Hmm... ok, maybe we should. I mean, it's a trivial issue and I doubt we'll get to it any time soon. On the other hand, even if it gets closed as stale eventually, maybe it's good to at least have a record that this is happening at all. It's just whitespace so it may not be apparent to people.

Reply:
In any case, this should not really block this PR. It was meant to be just a quick cleanup.

Now that I think of it - do you expect it will cause conflicts with your nlohmann JSON PR? If it does, I can help rebasing, as I offered in the description.

Reply:
Created an issue here: #15033. Just note that it's **very** low priority. No particular need to jump at it right now.